modules/op-pac-configuring-github-app-manually.adoc

```diff
@@ -0,0 +1,84 @@
+:_mod-docs-content-type: PROCEDURE
+[id="pac-configuring-github-app-manually_{context}"]
+= Configuring a GitHub App manually and creating a secret for {pac}
+
+You can use the GitHub user interface to create a GitHub app. Then you must create a secret that configures {pac} to connect to GitHub app.
+
+If you created additional {pac} controllers to support additional GitHub apps, you must use this procedure for the additional controllers.
+
+.Procedure
+. Sign in to your GitHub account.
+
+. in the GitHub menu, select **Settings** -> **Developer settings** -> **GitHub Apps**, then click **New GitHub App**.
+
+. Provide the following information in the GitHub App form:
+
+* **GitHub Application Name**: `{pipelines-shortname}`
+* **Homepage URL**: OpenShift Console URL
+* **Webhook URL**: The {pac} route or ingress URL. You can find it by running the following command:
++
+[source,terminal]
+----
+$ echo https://$(oc get route -n openshift-pipelines pipelines-as-code-controller -o jsonpath='{.spec.host}')
+----
++
+Alternatively, to configure the GitHub app for an additional {pac} controller, replace `pipelines-as-code-controller` with the name of the controller that you configured, as in the following example:
++
+[source,terminal]
+----
+$ echo https://$(oc get route -n openshift-pipelines pac_controller_2 -o jsonpath='{.spec.host}')
+----
+
+* **Webhook secret**: An arbitrary secret. You can generate a secret by running the following command:
++
+[source,terminal]
+----
+$ openssl rand -hex 20
+----
+
+. Select the following items in the **Repository permissions** section:
+
+* **Checks**: `Read & Write`
+* **Contents**: `Read & Write`
+* **Issues**: `Read & Write`
+* **Metadata**: `Read-only`
+* **Pull request**: `Read & Write`
+
+. Select the following items in the **Organization permissions** section:
+
+* **Members**: `Readonly`
+* **Plan**: `Readonly`
+
+. Select the following items in the **User permissions** section:
+
+* **Check run**
+* **Issue comment**
+* **Pull request**
+* **Push**
```

Comment:
can you update the list with this

```
Subscribe to following events:

* Check run
* Check suite
* Issue comment
* Commit comment
* Pull request
* Push
```
Ref : https://release-v0-24-1.pipelines-as-code.pages.dev/docs/install/github_apps/#manual-setup